samples/drivers/adc/boards/nucleo_h743zi.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_LOG=y
+CONFIG_ADC_LOG_LEVEL_DBG=y
+CONFIG_DMA_LOG_LEVEL_DBG=y
```

Comment:
I think this can be removed as well.

Reply:
I think only CONFIG_ADC_STM32_DMA=y is required here